### PR TITLE
Locales readme

### PR DESCRIPTION
**Pull request by habitullence**
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1367)

### CODE REVIEW

**Comment by tomas-stefano** [comment]
:+1:

File: config/locales/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Locales
+
+We didn't really have any conventions in place before for organising the locale files which made it hard to know where to put/find translations.
+
+The new convention is to have a one-to-one mapping. If we take the BabyCostCalculatorController as an example, in the deny_non_syndicates method we have a variable called ```controllers.baby_cost_calculator.deny_non_syndicates.article_id``` which be found in ```/config/locales/controllers/baby_cost_calculator.{locale}.yml```
```
Comment (munckymagik) [inline config/locales/README.md]: "...which *would* be..."?

**Comment by munckymagik** [comment]
:+1:

File: config/locales/README.md (hunk 1 above)
Comment (munckymagik) [inline config/locales/README.md]: Sorry folks going to add `[WIP]` to the title as there's work to do here, yet this still keeps appearing on MAS Bot's reminders.

**Comment by jongilbraith** [comment]
Rebased and amended the readme to address @munckymagik's comment.

Merging...

**Pull request merged**